security/interceptor/security-interceptor-anonymous/src/main/java/org/codice/ddf/security/interceptor/AnonymousInterceptor.java

```diff
@@ -286,15 +303,10 @@ public void handleMessage(SoapMessage message) throws Fault {
                                     String xpathStrict = "/Layout/Policy/Strict";
                                     String xpathLaxTimestampFirst = "/Layout/Policy/LaxTimestampFirst";
                                     String xpathLaxTimestampLast = "/Layout/Policy/LaxTimestampLast";
-                                    layoutLax = evaluateExpression(xml, xpathLax);
-                                    layoutStrict = evaluateExpression(xml, xpathStrict);
-                                    layoutLaxTimestampFirst = evaluateExpression(xml, xpathLaxTimestampFirst);
-                                    layoutLaxTimestampLast = evaluateExpression(xml, xpathLaxTimestampLast);
 
                                 } else if (qName.equals(SP12Constants.TRANSPORT_TOKEN)) {
                                 } else if (qName.equals(SP12Constants.HTTPS_TOKEN)) {
                                     String xpath = "/HttpsToken/Policy/RequireClientCertificate";
-                                    requireClientCert = evaluateExpression(xml, xpath);
```

Comment:
Looks like requireClientCert isn't needed anymore either and this else if block could be empty.